confignode/src/main/java/org/apache/iotdb/confignode/manager/ConfigManager.java

```diff
@@ -698,103 +697,65 @@ public TPermissionInfoResp checkUserPrivileges(
   }
 
   @Override
-  public TConfigNodeRegisterResp registerConfigNode(TConfigNodeRegisterReq req) {
+  public TSStatus registerConfigNode(TConfigNodeRegisterReq req) {
     // Check global configuration
     TSStatus status = confirmLeader();
 
     if (status.getCode() == TSStatusCode.SUCCESS_STATUS.getStatusCode()) {
-      TConfigNodeRegisterResp errorResp1 = checkConfigNodeRegisterResp(req);
-      if (errorResp1 != null) return errorResp1;
+      TSStatus errorStatus = checkConfigNodeGlobalConfig(req);
+      if (errorStatus != null) {
+        return errorStatus;
+      }
 
       procedureManager.addConfigNode(req);
-      return nodeManager.registerConfigNode(req);
```

Comment:
nodemanager does not registerConfigNode?

Reply:
The nodeManager.registerConfigNode was used to get the current PartitionRegionId and ConfigNodeList. However, the purpose of registerConfigNode is only to activate the addConfigNodeProcedure, thus we don't need to get these information anymore. Therefore, I just delete this redundant interface.